src/plots/rectanglePlot.ts

```diff
@@ -251,6 +253,50 @@ export module Plots {
       });
     }
 
+    /**
+     * Gets the Accessor for label.
+     */
+    public label(): AccessorScaleBinding<any, string>;
```

Comment:
Example of returning `AccessorScaleBinding` even when there's no `Scale`: `symbol()` (`Plots.Scatter`)

Example of not returning a binding: `interpolator()` (`Plots.Line`)

I think if there's no foreseeable case in which a `Scale` makes sense, we shouldn't return an `AccessorScaleBinding`.